### ISSUE TITLE
Building Specialization

### ISSUE THREAD

**Issue by Hakhaktak** [derpfacedirk/Goldiriath#19]
We (Baron_Minedox and I) were wondering what kind of (interactive) buildings we are going to need, and if so, what buildings always should be presented in a village, city and stuff. I suggest that we're going to think about a list the very next meeting, or even in the comment section, since this list is needed for builders.

**Comment by derpfacedirk** [comment]
right now i would say a crafting area and a market area

**Comment by Hakhaktak** [comment]
I think a tavern, leather tanner, armory, black smith, fletcher, library and grocery (for potions) are needed as well.

**Comment by derpfacedirk** [comment]
what would the function of an armory be? officialy its a weapon storage
leather tanner, blacksmith, fletcher and grocery I would count under market area
also a bank would be rather usefull
and a G.A.H.

**Comment by JeromSar** [comment]
@derpfacedirk What exactly is a G.A.H?

Additionally, perhaps a general store where you can buy common items and sell loot.

**Comment by derpfacedirk** [comment]
G.A.H. 
global auction house
you can sell loot anywhere, and maybe instead of making a specialized potion store we make a general store

**Comment by JeromSar** [comment]
@derpfacedirk Yeah, I'd agree with that, however, the selling rates in the general store should be much lower than the buying prices, stimulating players to compete with lower prices in the GAH. 

Additionally, I think that for some specialized items, you'd have to visit a specialized store (e.g. for bolts and arrows, you'd have to visit a fletcher).

**Comment by derpfacedirk** [comment]
your first point is always a standard. in general you shouldnt sell loot in the stores just in the GAH if you want more money for it

the general store i was refering to was the same kind you were refering to in your first comment
a general store will mainly sell potions or other common items. actual gear (arrows, weapons, armor) will have to be bought in the specialized store

**Comment by JeromSar** [comment]
@derpfacedirk @Hakhaktak How about a temple or two for worshiping the different kinds of religions, provided they exist.

**Comment by derpfacedirk** [comment]
@JeromSar  good point

**Comment by Breesboar** [comment]
Actually I would like to have a list of all the interactive buildings and a specific one. Just "shop" or "market area" is too little information for me as builder. I would like to hear grocery and fletcher and blacksmith that kind of things, like @Hakhaktak said. I can't make this list on my own because I don't have much experience when it comes to MMO's and I don't know what's possible in the plugin and what's not.
Me and Rik and maybe future builders will need this list every time we build a village or city. So is here someone who wants to make a complete list for us?

**Comment by derpfacedirk** [comment]
I'll get on it.

**Comment by cashenkes** [comment]
There is also the small point that not every city will need all the buildings, the bigger the city the more buildings there are. If you really want to have everything in one city go to the ''hub'' that is the reason it is there.

**Comment by Breesboar** [comment]
I know. Small villages will only have basic stuff like food stores. Specialized and more important interactive buildings will be located in cities. The number of those important buildings will be more or less the same for both factions to keep things fair.

**Comment by Hakhaktak** [comment]
@Breesboar It wouldn't matter that much if the one faction has more specialized buildings than the other since all of them will sell weapons that are equal to each other. It would only make the amount of places to get a weapon smaller. But I also do not see a reason why we should not balance it so yeah.. 

We have to talk about hubs btw. What kind of hubs are we going to make?

**Comment by derpfacedirk** [comment]
Hubs are cities that (usually) have all nessicary facility's
shops, crafting, quests. and where players meet up.
we cant do a lot about what city becomes a hub. that is up to the players
but we can make cities that are easy to reach and have all facility's and so the potential to become a hub

**Comment by Hakhaktak** [comment]
My idea of a hub is just a big market plaza where a market is, how surprising. The tradecenter(s) should be connected directly with this plaza and I think the more specialized shops should be in different streets. It'll let people explore the city's and look at their beauty and also a bit more realistic. The hub should only there for fast and simple trade and creating party's. F.e. Gathering a group to attack a fortress

**Comment by derpfacedirk** [comment]
basicly what we call the hub is an area (usually a city) where players meet up in game
for example in LOTRO when you agreed to a trade over global chat you would say: "come to bree."
because for some reason 50% of all player meetings happened in bree

**Comment by cashenkes** [comment]
As for the list that someone was going to make:
 basic villages would need, a general store, a blacksmith, a stable (for fast-traveling) and a food store (at least).
 Bigger city's would need the same base with Some extra's like a library and a fletcher.
